### DISCUSSION TITLE
Key Mismatch or Stuck Connecting

### DISCUSSION THREAD

**Discussion by RakanRocky** [rustdesk/rustdesk#10078] [Q&A]
When trying to connect to any machine we get a key mismatch or it just loading and saying it is connecting but does not connect. We verified the key and reloaded it but does not work.

**Comment by rustdesk** [comment]
Please https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, search there first, some users might have asked similar questions. Usually nobody will help you with build or self-hosting here.
